# MotionModeFeedback.msg - 运动模式反馈
# 运动模式反馈
uint8 mode              # 运动模式码
string mode_text        # 运动模式文本描述
uint8 switch_status     # 切换状态码
string switch_text      # 切换状态文本描述
uint8 drive_mode        # 驱动模式码
string drive_mode_text  # 驱动模式文本描述